velociraptor/autoplotter/compare.py

```diff
@@ -65,6 +65,13 @@ def load_yaml_line_data(
     for path, name in zip(paths, names):
         try:
             with open(path, "r") as handle:
+                inc = 0
```

Comment:
What exactly is the point of this bit of code? Is this a leftover from debugging?